include/tf-psa-crypto/private/crypto_adjust_config_support.h

```diff
@@ -0,0 +1,219 @@
+/**
+ * \file tf-psa-crypto/private/crypto_adjust_config_support.h
+ * \brief Adjust TF-PSA-Crypto configuration: support modules
+ *
+ * This is an internal header. Do not include it directly.
+ *
+ * Activate parts of support modules, based on the user configuration
+ * as well as requirements of generic code and requirements of
+ * driver-specific code.
+ */
+/*
+ *  Copyright The Mbed TLS Contributors
+ *  SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+ */
+
+#ifndef TF_PSA_CRYPTO_PRIVATE_CRYPTO_ADJUST_CONFIG_SUPPORT_H
+#define TF_PSA_CRYPTO_PRIVATE_CRYPTO_ADJUST_CONFIG_SUPPORT_H
+
+#if !defined(TF_PSA_CRYPTO_CONFIG_FILES_READ)
+#error "Do not include mbedtls/config_adjust_*.h manually! This can lead to problems, " \
```

Comment:
The "Do not include mbedtls/config_adjust_*.h" part should be updated.
```suggestion
#error "Do not include crypto_adjust_config_*.h manually! This can lead to problems, " \
```
?

Reply:
Actually I think this error is obsolete, now that the header is under `private`. We added this error because there used to be no indication that the header wasn't a public interface. So I'll remove it.